ESLint errors prevent new app from being generated

I am having the problem with following messages

```
yarn run v1.22.10
$ npm run -- lint:js --fix && npm run -- lint:css --fix

> lint:js
> stale-dep && eslint 'app/{components,frontend,javascript}/**/*.{js,jsx}' --fix


Oops! Something went wrong! :(

ESLint: 8.43.0

Error: Cannot find module 'word-wrap'
Require stack:
- /Users/twnaing/Area-51/rails-templates-m2/node_modules/optionator/lib/help.js
- /Users/twnaing/Area-51/rails-templates-m2/node_modules/optionator/lib/index.js
- /Users/twnaing/Area-51/rails-templates-m2/node_modules/eslint/lib/options.js
- /Users/twnaing/Area-51/rails-templates-m2/node_modules/eslint/lib/cli.js
- /Users/twnaing/Area-51/rails-templates-m2/node_modules/eslint/bin/eslint.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1070:15)
    at Module._load (node:internal/modules/cjs/loader:923:27)
    at Module.require (node:internal/modules/cjs/loader:1137:19)
    at require (node:internal/modules/helpers:121:18)
    at Object.<anonymous> (/Users/twnaing/Area-51/rails-templates-m2/node_modules/optionator/lib/help.js:6:14)
    at Object.<anonymous> (/Users/twnaing/Area-51/rails-templates-m2/node_modules/optionator/lib/help.js:260:4)
    at Module._compile (node:internal/modules/cjs/loader:1255:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1309:10)
    at Module.load (node:internal/modules/cjs/loader:1113:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I have nodes `20.2.0`, yarn `1.22.10` and the package.json has this.

```json
{
  "private": "true",
  "scripts": {
    "fix": "npm run -- lint:js --fix && npm run -- lint:css --fix",
    "lint": "npm run lint:js && npm run lint:css",
    "lint:js": "stale-dep && eslint 'app/{components,frontend,javascript}/**/*.{js,jsx}'",
    "lint:css": "stale-dep && stylelint 'app/{components,frontend,assets/stylesheets}/**/*.{css,scss}'",
    "postinstall": "stale-dep -u"
  },
  "dependencies": {
    "eslint": "^8.43.0",
    "eslint-config-prettier": "^8.8.0",
    "eslint-plugin-prettier": "^4.2.1",
    "postcss": "^8.4.24",
    "prettier": "^2.8.8",
    "stale-dep": "^0.6.0",
    "stylelint": "^15.9.0",
    "stylelint-config-recommended-scss": "^12.0.0",
    "stylelint-config-standard": "^33.0.0",
    "stylelint-declaration-strict-value": "^1.9.2",
    "stylelint-prettier": "^3.0.0",
    "stylelint-scss": "^5.0.1"
  }
}

```

--- a/template.rb
+++ b/template.rb
@@ -249,7 +249,7 @@ def add_yarn_lint_and_run_fix
   packages = %w[
     eslint
     eslint-config-prettier
-    eslint-plugin-prettier
+    eslint-plugin-prettier@^5.0.0-alpha.2
     postcss
     prettier
     stale-dep
@@ -263,7 +263,7 @@ def add_yarn_lint_and_run_fix
   add_package_json_script("lint:js": "stale-dep && eslint 'app/{components,frontend,javascript}/**/*.{js,jsx}'")
   add_package_json_script("lint:css": "stale-dep && stylelint 'app/{components,frontend,assets/stylesheets}/**/*.css'")
   add_package_json_script("postinstall": "stale-dep -u")
-  run_with_clean_bundler_env "yarn add #{packages.join(' ')}"
+  run_with_clean_bundler_env "yarn add #{packages.map(&:shellescape).join(' ')}"
   run_with_clean_bundler_env "yarn fix"
 end
 

